modules/ding_ting_frontend/ding_ting_frontend.info

```diff
@@ -19,7 +19,7 @@ dependencies[] = page_manager
 dependencies[] = pm_existing_pages
 dependencies[] = strongarm
 dependencies[] = ting
-dependencies[] = ting_covers
```

Comment:
Changing the dependency to `ting_covers_addi` is problematic. It compares to adding a dependency to a provider implementation like `alma`.

- The dependency is misleading. `ding_ting_frontend` depends on `ting_covers` - not the addi covers provider.
- It will in effect prevent sites from disabling the addi integration even though they want to use some other cover provider.

If the module should be enabled by default on install then it should be added to `ding_ting_frontend_install()`.